spec/jobs/proposals_closing_soon_job_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+describe ProposalsClosingSoonJob do
+  let!(:motion) { create :motion, closing_at: 24.hours.from_now }
+  let(:subject) { ProposalsClosingSoonJob.new.perform }
+
+  describe 'perform' do
+    it 'publishes a proposal closing soon event' do
```

Comment:
I think the tests spell out the intended functionality nicely.

- publishes a proposal closing soon event successfully
- does not publish for closed motions
- does not publish for motions that aren't closing soon
- does not publish for motions that just had a motion_closing_soon published
- publishes if a motion_closing_soon event was already published, but more than 2 days ago.